docs/pages/management/admin/troubleshooting.mdx

```diff
@@ -27,8 +27,17 @@ To diagnose problems, you can configure the `teleport` process to run with
 verbose logging enabled by passing it the `-d` flag. `teleport` will write logs
 to stderr.
 
-Logs will resemble the following (these logs were printed while joining a Node
-to a cluster, then terminating the `teleport` process on the Node):
+Alternatively, you can set the log level from the Teleport configuration file:
+
+```yaml
+teleport:
+  log:
+    severity: DEBUG
+```
+
+Restart the `teleport` process to apply the modified log level. Logs will resemble
+the following (these logs were printed while joining a Node to a cluster, then
+terminating the `teleport` process on the Node):
```

Comment:
```suggestion
Restart the `teleport` process to apply the modified log level. Logs will resemble
the following (these logs were printed while joining a server to a cluster, then
terminating the `teleport` process on the server):
```
Since we're not using the "Node" concept anymore